tests/db/test_bigchain_api.py

```diff
@@ -1760,13 +1789,16 @@ def test_create_and_fulfill_asset_with_hashlock_condition(self, b, user_vk):
         assert b.validate_transaction(hashlock_tx_signed) == hashlock_tx_signed
         assert b.is_valid_transaction(hashlock_tx_signed) == hashlock_tx_signed
 
-        b.write_transaction(hashlock_tx_signed)
-
         # create and write block to bigchain
         block = b.create_block([hashlock_tx_signed])
         b.write_block(block, durability='hard')
 
-        assert len(b.get_owned_ids(b.me)) == 0
+        # vote block valid
+        vote = b.vote(block['id'], b.get_last_voted_block()['id'], True)
+        b.write_vote(vote)
+
+        # this assertion must be wrong...?
+        # assert len(b.get_owned_ids(b.me)) == 0
```

Comment:
yeah, it was to see that we didnt own anything yet before redeeming the hashlock. But a bit of a stupid test